benchmarks/capnproto_decode/bench.lua

```diff
@@ -0,0 +1,50 @@
+local capnp = require "capnp"
+local ffi = require "ffi"
+local example_capnp = require "example_capnp"
+
+local list_size = 100
+local list = {}
+
+for i=1,list_size do

```

Comment:
further down in the file, you have spaces after your commas. I suggest you choose one and stick with it.